lib/jwt.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { promisify } = require('util');
+const jwt = require('jsonwebtoken');
+const redis = require('redis');
+
+const User = require('models/User');
+
+const {
+    REDIS,
+    JWT
+} = require('config/config');
+
+module.exports = {
+    verify,
+    generate,
+    remove
+};
+
+async function verify (token) {
+    const redis_client = redis.createClient({
+        host: REDIS.host,
+        port: REDIS.port,
+        prefix: REDIS.prefix
+    });
+
+    const sismemberAsync = promisify(redis_client.sismember).bind(redis_client);
```

Comment:
In other places snake case is used as a variable name.

Reply:
Made changes based on the comment.